src/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -39,6 +39,20 @@ By default, it'll mask all fields, so no personally identifiable information (PI
 
 <PlatformContent includePath="session-replay/setup" />
 
+### Content Security Policy (CSP)
+Session Replay uses a WebWorker to perform work (e.g. compression) off the main UI thread so as not to degrade the performance of your application. Add the below entry to make sure that workers can be loaded: 
+
+
+```
+worker-src 'self' blob:
+```
+
+Older versions of Safari do not support `worker-src`, you will need to additionally add an entry for `child-src`:
```

Comment:
Do we know what versions exactly? As a reader I'm confused: Well I have users on Safari so do I need to add this too?